app/controllers/active_storage/blobs_controller.rb

```diff
@@ -37,15 +37,14 @@ def check_user
     if current_user.nil?
       redirect_to new_user_session_path
     else
-      # current_organisation is not available here
-      org = Organisation.find(
-        ActiveStorage::Attachment
-          .find_by(blob: @blob)
-          .record_id
-      )
-
-      unless can? :read_mou, org
-        redirect_to overview_index_path, alert: "You are not allowed to see this MoU."
+      attachment = ActiveStorage::Attachment.find_by(blob: @blob)
+
+      if attachment.name == "signed_mou"
```

Comment:
Another comment: we don't actually have a test that covers normal admins downloading the template. But we have one for the super admin which covers this enough so, again, I am not bothered if you don't add one.

Reply:
I've added that now.